conda-environment.yml

```diff
@@ -17,9 +17,9 @@ dependencies:
   - python
   - pyzmq >=22,<23
   - setuptools >=49, <67
+  - importlib_metadata # [py<3.8]
   - urwid >=2,<3
-  # Add # [py<3.11] for tomli once Python 3.11 Released
-  - tomli >=2
+  - tomli >=2 # [py<3.11]
 
```

Comment:
I am not convinced these comments actually do anything. I can't find any reference to an equivalent to setuptool's environment markers for conda environment YML files online.

Actually, what is this file used for? Is it just CI?

Reply:
> I am not convinced these comments actually do anything. I can't find any reference to an equivalent to setuptool's environment markers for conda environment YML files online.

https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#preprocessing-selectors

> Actually, what is this file used for? Is it just CI?

This file mirrors the setup.cfg with Conda, it currently has two uses:

* Defining the environment for remote tests.
* Updating the conda-forge feedstock.

We should do more testing with this environment.